internal/provider/schema_util.go

```diff
@@ -163,23 +163,22 @@ func GetAutomatedRootRotationSchema() map[string]*schema.Schema {
 		consts.FieldRotationSchedule: {
 			Type:        schema.TypeString,
 			Optional:    true,
-			Description: "Schedule",
+			Description: "The cron-style schedule for the root credential to be rotated on. Cannot be used with rotation_period.",
 		},
 		consts.FieldRotationPeriod: {
 			Type:        schema.TypeInt,
 			Optional:    true,
-			Description: "Period",
+			Description: "The period of time in seconds between each rotation of the root credential. Cannot be used with rotation_schedule.",
 		},
 		consts.FieldRotationWindow: {
 			Type:        schema.TypeInt,
 			Optional:    true,
-			Description: "Window",
+			Description: "The maximum amount of time in seconds Vault is allowed to complete a rotation once a scheduled rotation is triggered. Can only be used with rotation_schedule.",
 		},
 		consts.FieldDisableAutomatedRotation: {
 			Type:        schema.TypeBool,
 			Optional:    true,
-			Computed:    true,
```

Comment:
Confirmed with AWS Secrets that removing Computed fixes the error where removing the field was not being picked up in the diff. Also looks like default values are being adhered to. Thanks for the fix!